sentry-ruby/lib/sentry/rack/capture_exceptions.rb

```diff
@@ -17,23 +17,23 @@ def call(env)
           scope.set_rack_env(env)
 
           transaction = start_transaction(env, scope)
-          scope.set_span(transaction)
+          transaction ? scope.set_span(transaction) : nil
 
           begin
             response = @app.call(env)
           rescue Sentry::Error
-            finish_transaction(transaction, 500)
+            transaction ? finish_transaction(transaction, 500) : nil
```

Comment:
this and the below change are unnecessary with the early return in `finish_transaction`

Reply:
Ah, right - good point.